cmd/openshift/operator/kodata/openshift/tekton-results-route-rbac/rbac.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
do we need `Role` or `ClusterRole`?

Reply:
We want to `aggregate-to-view`  accounts also. That's why I used ClusterRole. We can use ClusterRole in Rolebinding.